applications/asset_tracker/Kconfig

```diff
@@ -153,6 +153,15 @@ endif
 
 endmenu # Device
 
+config MOTION
```

Comment:
How about making this `menuconfig MOTION` so that we can access the submenu directly from the `MOTION` option? See how it's don with `LIGHT_SENSOR` for instance:
![image](https://user-images.githubusercontent.com/22167799/100238722-d0073100-2f30-11eb-88b2-69053f175f51.png)

Reply:
thanks for the suggestion! fixed it now